src/modules/flight_mode_manager/tasks/FlightTask/FlightTask.cpp

```diff
@@ -161,6 +162,24 @@ void FlightTask::_evaluateVehicleLocalPosition()
 	}
 }
 
+void FlightTask::_evaluateVehicleLocalPositionSetpoint()
+{
+	_velocity_setpoint_feedback.setAll(NAN);
```

Comment:
It might be slightly better to avoid setting to NAN every iteration then resetting to something else. How about only update below, or set to NAN if the timeout is exceeded?

Reply:
Makes total sense. I changed it. Maybe it would also make sense for the other topics even though there's partial validity there.

Reply:
There it's clearer to set NAN at the beginning compared to handling each and every case separately.